### ISSUE TITLE
Cross-Origin issue

### ISSUE THREAD

**Issue by josefsj3m** [WWBN/AVideo#3285]
Hi all, when I request a video from an AVideo installed on my server using the embed code it plays ok (it uses an iframe). But when it is requested thru javascript using data-src I get a cross-origin read blocking error from chrome.

I am using this, Header set Access-Control-Allow-Origin "*", in the Apache virtual host conf file, but chrome still refuses to get the video.

I'll appreciate some help.

Thanks

**Comment by akhilleusuggo** [comment]
have should remove it from the apache2 conf . I believe you're getting the duplicated error on javascript `Header set Access-Control-Allow-Origin "*"`

Provide some lscreenshot of javacript , also take a look at this issue .
https://github.com/WWBN/AVideo/issues/3072

1-Remove it from apache2 OR
2-Comment the lines on the script .

**Comment by josefsj3m** [comment]
I think the issue is not related to a duplicated Header set Access-Control-Allow-Origin "*", because I removed it to test and the error still remains.

I put the same video using jquery fancybox, that requests the video with javascript, and using the embed code from the AVideo installation. Video 1 uses fancybox, Video 2 uses embed code, there is no issue for video 2. I uploaded the same video to Vimeo and played it with fancybox with no problem. So, I think the problem is related to an incorrect configuration on my AVideo/Apache server.

Please, take a look at the below screenshots. 

![screenshot1](https://user-images.githubusercontent.com/5265061/85731011-8770f080-b6d0-11ea-92cb-d5f3f140c638.png)

When I click on the video 1 I get the CORS issue as you can see below:

![screenshot2](https://user-images.githubusercontent.com/5265061/85731294-c69f4180-b6d0-11ea-936c-ff31aff66720.png)

You can see below the headers when the error occurs.

![screenshot3](https://user-images.githubusercontent.com/5265061/85732793-0adf1180-b6d2-11ea-877f-ec744ec8360a.png)

Also, there is a second problem, but not related to CORS. When I use the embed code from the AVideo I got a javascript error inside the iframe, you can see it in red on the screenshots above. It tries to close a navbar that doesn't exist, but the video plays without problem.

I'd like to use the jquery fancybox, but I am almost giving up. 

Thanks for your attention.

**Comment by akhilleusuggo** [comment]
Did you restarted apache2 and php after your removed the CORS from apache ?
Also I may think this could have something to do with .htaccess 

Have you added any other header functions ? X-Frame-Options , X-Content-Type-Options , X-XSS-Protection , X-Permitted-Cross-Domain-Policies , etc etc etc ?

**Comment by josefsj3m** [comment]
Yes, I restarted Apache/PHP. I didn't add any X options to the .htaccess.

**Comment by josefsj3m** [comment]
The CORS problem was resolved, I used Firefox instead of Chrome to test and the problem did not occur in Firefox. I reinstalled Chrome and the problem was gone. 

But, the video still doesn't appear using fancybox (no CORS issue now). I used a copied url when I right-click on the video. But, I realized that what AVideo sends is an html and not a streaming, I think this is preventing the video from loading through fancybox. How can I get the video url?
